### PR TITLE
Update default catalog 8.2.x

### PR DESCRIPTION
**Pull request by ps-jarvis**
| Questions         | Answers
| ----------------- | ------------------------------------------------------------
| Branch?           | 8.2.x
| Description?      | Automatic update of default catalog for 8.2.x
| Type?             | improvement
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/10940726730
| Fixed ticket?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Review by matthieu-rolland — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by florine2623** [comment]
@jolelievre tests are green, no need functional QA on this ^^

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
